src/test/unit/utils/RandomTest.cpp

```diff
@@ -414,7 +414,7 @@ void RandomTest::RunTests()
 
     {
       // choose 0 elements
-      UInt32 choices[0];
+      UInt32 choices[1];
```

Comment:
Yes. MSVC argues that `the size of an array must be greater than zero`. I will remove it then.